src/send-commercial-metrics.spec.ts

```diff
@@ -288,66 +287,64 @@ describe('send commercial metrics code', () => {
 });
 
 describe('send commercial metrics helpers', () => {
-	const eventProperties = {
-		type: undefined,
-		downlink: 1,
-		effectiveType: '4g',
-	};
-
-	const transformedProperties: Array<[string, string | number | undefined]> =
-		[
+	it('can transform event timer properties into object entries', () => {
+		expect(
+			transformToObjectEntries({
```

Comment:
I'm a fan of inlining the function calls and expected values like this - cuts down on jumping around the file to find these values.